discussions/models.py

```diff
@@ -104,3 +105,12 @@ class Comment(Content):
     @queryset_manager
     def objects(doc_cls, queryset):
         return queryset(_type='Comment')
+
+
+class Subscriptions(Document):
+    _id = ObjectIdField(required=True, primary_key=True)
+    subscriber_id = StringField(required=True)  # This is a int string -.-
+    source_id = StringField(required=True)  # This is a int string -.-
+    source_type = StringField(required=True)  #TODO Can this be a ChoiceField?
+    updated_at = DateTimeField(required=True)  # TODO is this DateTime
```

Comment:
Nit: refactor created/updated at into abstract class that can be shared by all the models that use it, since it seems like a pattern.